conan/tools/meson/toolchain.py

```diff
@@ -396,6 +396,14 @@ def _sanitize_format(v):
         # These link_args have already the LDFLAGS env value so let's add only the new possible ones
         self.objc_link_args.extend(apple_flags + extra_flags["ldflags"])
         self.objcpp_link_args.extend(apple_flags + extra_flags["ldflags"])
+        # Meson properties
+        sys_root = self._conanfile.conf.get("tools.build:sysroot", check_type=str)
+        if sys_root:
+            self.properties["sys_root"] = sys_root
+            # self.c_args.append("--sysroot=" + sys_root)
+            # self.cpp_args.append("--sysroot=" + sys_root)
+            # self.c_link_args.append("--sysroot=" + sys_root)
+            # self.cpp_link_args.append("--sysroot=" + sys_root)
```

Comment:
These are then unnecessary, and Meson will manage with the property? (recall to remove this code, you can put a comment like ``# property sys_root is enough, not necessary to define cxx flags``